CONTRIBUTING.md

```diff
@@ -309,6 +309,25 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+After the original PR was merged into master, backporting is done only
+when necessary - the fix either needs to be present in a release branch
+(e.g. a security issue) or on request.
```

Comment:
```suggestion
(e.g., a security issue) or on request.
```

Reply:
Thank you. A nice one. I actually learned something.
https://www.dailywritingtips.com/comma-after-i-e-and-e-g/